spec/factories/planned_disbursement.rb

```diff
@@ -11,5 +11,7 @@
     receiving_organisation_name { Faker::Company.name }
     receiving_organisation_reference { "GB-COH-{#{Faker::Number.number(digits: 6)}}" }
     receiving_organisation_type { "70" }
+
+    association :report
```

Comment:
Won't this automatically create an associated report and anything using the trait `with_report` will create an additional one..?

Reply:
Possibly; however in (e.g.) spec/features/staff/users_can_create_a_planned_disbursement_spec.rb:57 the Planned Disbursement isn't being created with a factory, it's being created through the UI, so I need to use a `project_activity` `:with_report` to make sure a Report is available for the Planned Disbursement to attach to. IYSIM?

Reply:
I see your point, although I'd probably manually create the report and then reference it in `project_activity` in this case.

Just trying to avoid having tests that are creating unnecessary associations - for example, if you created several different planned disbursements, by default they'd each be associated with a brand new project, which doesn't really reflect what would happen in reality.

Reply:
Ah OK I didn't realise what you were getting at! I think I was labouring under an assumption that Factories needed to reflect the associations of their models, hence I put it in.

Reply:
Although it turns out if I remove the association, all my PlannedDisbursementPolicy tests fail